server/src/main/scala/manifests/ManifestLookup.scala

```diff
@@ -66,6 +71,24 @@ case class ManifestLookup(tables: Tables)
     }
   }
 
+  def manifestPaxForScheduled(flightKeys: Vector[(String, String, String, Timestamp)])
+                             (implicit mat: Materializer): Future[Int] = {
+    val eventualMaybePaxProfiles = Source(paxForArrivalQuery(flightKeys))
+      .mapAsync(1)(paxProfilesFromQuery)
+      .withAttributes(StreamSupervision.resumeStrategyWithLog(getClass.getName))
+      .runWith(Sink.seq)
+
+    val paxProfilesCount = eventualMaybePaxProfiles.map { tries =>
+      tries.collect {
+        case Failure(t) => log.warn(s"Failed to get manifests", t)
+      }
+      tries.collect { case Success(paxProfiles) => paxProfiles }
+    }.map(_.map(_.toSet.size))
```

Comment:
It's quite difficult to see what this line is doing. Can you break it out into some named variables so we can see what the data is?